src/main/java/pl/edu/mimuw/queue/FIFOIntQueue.java

```diff
@@ -1,5 +1,42 @@
 package pl.edu.mimuw.queue;
 
 public class FIFOIntQueue extends AbstractIntQueue {
-  // TODO: make changes with this class fields, constructors, methods
+
+  private IntQueueNode last;
+
+  public FIFOIntQueue() {
+    super();
```

Comment:
domyślnie wołane, raczej nie widziałem konwencji wołania bezparametrowych this/super